internal/k8s/status.go

```diff
@@ -259,13 +261,22 @@ func getExternalServiceAddress(svc *api_v1.Service) []string {
 func (su *statusUpdater) SaveStatusFromExternalStatus(externalStatusAddress string) {
 	su.externalStatusAddress = externalStatusAddress
 	if externalStatusAddress == "" {
-		// if external-status-address was removed from configMap, fall back on
-		// external service if it exists
+		// if external-status-address was removed from configMap
+
+		// fall back on external service if it exists
 		if len(su.externalServiceAddresses) > 0 {
 			su.saveStatus(su.externalServiceAddresses)
 			su.externalEndpoints = su.generateExternalEndpointsFromStatus(su.status)
 			return
 		}
+
+		// fall back on NginxCisConnector if it exists
+		if len(su.bigIPAddress) > 0 {
+			ips := []string{su.bigIPAddress}
+			su.saveStatus(ips)
```

Comment:
I think it's fine to inline these if you want:
```suggestion
			su.saveStatus([]string{su.bigIPAddress})
```